src/main/kotlin/frc/team5499/frc2018Kotlin/auto/actions/ArmAction.kt

```diff
@@ -0,0 +1,69 @@
+package frc.team5499.frc2018Kotlin.auto.actions
+
+import frc.team5499.frc2018Kotlin.subsystems.Arm
+
+public class ArmAction(armDirectionSt: ArmAction.ArmDirection, intakeDirectionSt: ArmAction.IntakeDirection, timeout: Double) : Action(timeout) {
+
+    enum class ArmDirection {
+        UP,
+        DOWN,
+        HOLD_UP,
+        HOLD_DOWN,
+        NONE
+    }

```

Comment:
You can make this an enum class with a value assigned to it.
`
enum class ArmDirection(val speed: Double) {
UP(Constants.Arm.UP_SPEED),
...
}
`